# oteodoro changes:
# Patch status:  testing
# For use for the distro's hardened profile

License header for include/clang/Driver/Options.td:

//===--- Options.td - Options for clang -----------------------------------===//
//
// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
// See https://llvm.org/LICENSE.txt for license information.
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
//
//===----------------------------------------------------------------------===//

diff -urp clang.orig/include/clang/Driver/Options.td clang/include/clang/Driver/Options.td
--- clang.orig/include/clang/Driver/Options.td	2021-08-26 11:45:39.000000000 -0700
+++ clang/include/clang/Driver/Options.td	2021-09-28 18:54:19.248936780 -0700
@@ -5557,7 +5557,7 @@ def stack_protector : Separate<["-"], "s
   HelpText<"Enable stack protectors">, Values<"0,1,2,3">,
   NormalizedValuesScope<"LangOptions">,
   NormalizedValues<["SSPOff", "SSPOn", "SSPStrong", "SSPReq"]>,
-  MarshallingInfoEnum<LangOpts<"StackProtector">, "SSPOff">;
+  MarshallingInfoEnum<LangOpts<"StackProtector">, "SSPStrong">;		// Changed by oteodoro for documentation consistency
 def stack_protector_buffer_size : Separate<["-"], "stack-protector-buffer-size">,
   HelpText<"Lower bound for a buffer to be considered for stack protection">,
   MarshallingInfoInt<CodeGenOpts<"SSPBufferSize">, "8">;
